plugins/modules/template_workflow_manager.py

```diff
@@ -2745,20 +2745,38 @@ def verify_diff_merged(self, config):
                 return self
 
             self.get_have_template(config, is_template_available)
-            self.log("Current State (have): {0}".format(self.want.get("template_params")), "INFO")
-            self.log("Desired State (want): {0}".format(self.have_template.get("template")), "INFO")
+            self.log("Desired State (have): {0}".format(self.want.get("template_params")), "INFO")
+            self.log("Current State (want): {0}".format(self.have_template.get("template")), "INFO")
             if not self.have_template.get("template"):
                 self.msg = "No template created with the name '{0}'".format(self.want.get("template_params").get("name"))
                 self.status = "failed"
                 return self
 
             template_params = ["language", "name", "projectName", "softwareType",
                                "softwareVariant", "templateContent"]
+            error_msg = "Configuration Template config is not applied to the Cisco Catalyst Center."
+            have_template = self.have_template.get("template")
+            want_template = self.want.get("template_params")
             for item in template_params:
-                if self.have_template.get("template").get(item) != self.want.get("template_params").get(item):
-                    self.msg = "Configuration Template config is not applied to the Cisco Catalyst Center."
+                if have_template.get(item) != want_template.get(item):
+                    self.msg = error_msg
```

Comment:
Can we specify the template name? Also why does Configuration Template config not applied to the Cisco Catalyst Center. Can we specify the reason for this failing?